### DISCUSSION TITLE
Error using CellExplorer

### DISCUSSION THREAD

**Discussion by GuilleSGCUPO** [petersenpeter/CellExplorer#76] [Q&A] [answered]
Hello guys, my name is Guillermo Sánchez-Garrido Campos, a PhD tudent from Seville (Spain). I am currently using Neuropixels and I wanted to use CellExplorer app after the Neuropixels course that took place last week. However, when I tried to run the "example-data" in MATLAB (R2019a) I got this error:
 
````
>> CellExplorer('metrics',cell_metrics);
[24-10-2022 18:04:12] Initializing session
[24-10-2022 18:04:12] Initializing dimensionality reduction plot
Error using CellExplorer>@(X)X(1:end-2)
Too many input arguments.

Error in CellExplorer/initializeSession (line 11267)
            customPlotOptions = cellfun(@(X) X(1:end-2),customPlotOptions.m,'UniformOutput', false);

Error in CellExplorer (line 217)
    initializeSession
````

I have followed the guidelines from the Github page, may anyone has had this issue before and knows how to fix it?
Thank you all in advance,

Guille

**Comment by petersenpeter** [comment, marked as answer]
Hi Guille

I don't recall that I have seen that error before. Could you check that you added all the folders from the CellExplorer GitHub repo to your Matlab set path?
